lisa/operating_system.py

```diff
@@ -820,7 +820,7 @@ def _install_packages(
                 f"dpkg -i {' '.join(file_packages)}", sudo=True, timeout=timeout
             )
             # after install package, need update the repo
-            self._initialize_package_installation()
+            self._initialize_package_installation(timeout=1800)
```

Comment:
there is no parameter timeout for method _initialize_package_installation

Reply:
Yes i removed it new push.